selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/dashboard/WorkspacesListTest.java

```diff
@@ -0,0 +1,200 @@
+  @Inject private TestWorkspaceServiceClient workspaceServiceClient;
+  @Inject private TestUser defaultTestUser;
+  @Inject private Workspaces workspaces;
+  @Inject private CreateWorkspace createWorkspace;
+  @Inject private ProjectSourcePage projectSourcePage;
+
+  @BeforeClass
+  public void setUp() throws Exception {
+    dashboard.open();
+
+    createWorkspace(workspaceName1);
+    createWorkspace(workspaceName2);
+    createWorkspace(workspaceName3);
+  }
+
+  @AfterClass
+  public void tearDown() throws Exception {
+    workspaceServiceClient.delete(workspaceName1, defaultTestUser.getName());
+    workspaceServiceClient.delete(workspaceName2, defaultTestUser.getName());
+    workspaceServiceClient.delete(workspaceName3, defaultTestUser.getName());
+  }
+
+  @BeforeMethod
+  public void openWorkspacesPage() {
+    dashboard.waitDashboardToolbarTitle();
+    dashboard.selectWorkspacesItemOnDashboard();
+  }
+
+  @Test
+  public void checkWorkspacesList() {
+    // check UI views of workspaces list
+    workspaces.waitToolbarTitleName();
+    workspaces.waitDocumentationLink();
+    workspaces.waitAddWorkspaceButton();
+    workspaces.waitSearchWorkspaceByNameField();
+    workspaces.waitBulkCheckbox();
+
+    // check all headers are present
+    ArrayList<String> headers = workspaces.getWorkspaceListHeaders();
+    assertTrue(headers.contains("NAME"));
+    assertTrue(headers.contains("RAM"));
+    assertTrue(headers.contains("PROJECTS"));
+    assertTrue(headers.contains("STACK"));
+    assertTrue(headers.contains("ACTIONS"));
+
+    // check workspaces info
+    Assert.assertEquals(workspaces.getWorkspaceRamValue(workspaceName1), "2048 MB");
+    Assert.assertEquals(workspaces.getWorkspaceStackName(workspaceName2), "java-default");
+    Assert.assertEquals(workspaces.getWorkspaceProjectsValue(workspaceName3), "1");
+  }
+
+  @Test
+  public void checkWorkspaceSelectingByCheckbox() {
+    // select the test workspace by checkbox and select it is checked
+    workspaces.selectWorkspaceByCheckbox(workspaceName1);
+    assertTrue(workspaces.isWorkspaceChecked(workspaceName1));
+    workspaces.selectWorkspaceByCheckbox(workspaceName1);
+    Assert.assertFalse(workspaces.isWorkspaceChecked(workspaceName1));
+
+    // click on the Bulk button and check that all workspaces are checked
+    workspaces.selectAllWorkspacesByBulk();
+    assertTrue(workspaces.isWorkspaceChecked(workspaceName1));
+    assertTrue(workspaces.isWorkspaceChecked(workspaceName2));
+    assertTrue(workspaces.isWorkspaceChecked(workspaceName3));
+
+    workspaces.selectAllWorkspacesByBulk();
+    Assert.assertFalse(workspaces.isWorkspaceChecked(workspaceName1));
+  }
+
+  @Test
+  public void checkWorkspaceActions() {
+    // open the Config page of the test workspace
+    workspaces.clickOnWorkspaceConfigureButton(workspaceName1);
+    workspaceDetails.waitToolbarTitleName(workspaceName1);
+    workspaceConfig.waitConfigForm();
+    seleniumWebDriver.navigate().back();
```

Comment:
going back looks strange here. What does this step test? If it helps to check on workspace name appearance, it's better to go to workspace list by direct link in Dashboard.